artifacts/chart/_crds/cluster.karmada.io_clusters.yaml

```diff
@@ -0,0 +1,314 @@
+
```

Comment:
how these crd yamls are generated?How are they synced with crds under artifacts/deploy? And I like ` artifacts/chart/crds/` rather than  ` artifacts/chart/_crds/`

Reply:
Default. helm will install the crd/ files in the current path to the current cluster, but in karmada, we should install the crd to the kube-apiserver

Reply:
> The karmada installation using helm should support two scenarios: with karmada-apiserver and without.And also should take cert-manager for CA injection into consideration.

Thanks. Can you describe in detail the situation of not installing the karmada-apiserver?

Reply:
> Default. helm will install the crd/ files in the current path to the current cluster, but in karmada, we should install the crd to the kube-apiserver

Yes. Clearly described in the document. https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

Reply:
> The karmada installation using helm should support two scenarios: with karmada-apiserver and without.And also should take cert-manager for CA injection into consideration.

> Thanks. Can you describe in detail the situation of not installing the karmada-apiserver?

Let's discuss it at the community meeting, a lot of background is involved.
For short answer, some users don't like to maintain an extra `karmada-apiserver` and `etcd`, they reused the one in clusters. 

I'm not sure we can do that in one chart. Personally, I want the chart to get onboard quickly, and keep iterating it to cover more cases.

Reply:
> how these crd yamls are generated?How are they synced with crds under artifacts/deploy? And I like ` artifacts/chart/crds/` rather than ` artifacts/chart/_crds/`

![image](https://user-images.githubusercontent.com/39866845/132481272-874d926b-3bca-4b83-bbd8-db12b3748080.png)
we should install crds in karmada-apiserver rather than kube-apiserver. So we use _crds Instead of crds.